docs/security_checks_guide.adoc

```diff
@@ -0,0 +1,56 @@
+= Verifying Self-Defence Security Check Results utilizing the metrics service
+
+This guide shows how to use Security Checks metrics to gather valueable information regarding the shape of the environment users run your mobile apps in, in relation to security risks that can lead to attack vectors against the data that is stored in-device.
+
+[discrete]
+== Prerequisites
+
+* An App that is using the AeroGear SDK (link TODO)
+* A running Metrics Service (link TODO)
+* An OpenShift account that will provide access to Grafana in the Metrics Service (link TODO)
+
+[discrete]
+== Accessing visualisations
+
+The visualisation for Security Checks metrics is done via set of Grafana dashboards, which in turn query the datasources for Security Checks metrics.
+
+The visualisations are spread across different dashboards:
+
+* Mobile Services Dashboard (Summary data in the 'Mobile Security Metrics' section)
+* Mobile Security Metrics Dashboard
+
+In general, the summary data that is visible on the 'Mobile Services' Dashboard is also available on the 'Mobile Security Metrics' dashboard, but in more detail.
+
+[discrete]
+== Analyzing Security Check Passing totals
+
+At the top level of the summary, general passing percentages for checks and totals regarding the amount of checks run are shown.
+
+These indicate the overall secureness of the environments your apps are running in, however since these totals are aggregated across all apps, platforms and clients a smaller overall percentage of passing checks does not imply that the clients are insecure as a whole, but can instead reflect a small amount of frequent users on a particularly insecure setup.
+
+[discrete]
+== Analyzing Failures per Check Type
+
+Following the above totals it is possible to see the amount of failures per type of check. These checks can vary in their scope and how critical they are to your App.
+Refer to the mobile app SDK documentation for details on each type of Security Check and implications of them failing.
```

Comment:
Is there a doc we could link to that explains what the different security check types are?

Reply:
I think the best we have right now is https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/getting-started/auth-self-defence-checks.adoc#provided-self-defence-checks

I'm thinking of linking it now, but not sure how these are going to be shared across the different platform sdks.